docs/reference/authentication.md

```diff
@@ -14,6 +14,22 @@ We recommend using basic authentication and a strong password to protect the `/s
 
 The API Gateway as of version 0.8.2 provides built-in basic authentication. To use it set the environmental variable `basic_auth` to true. Then create two secrets named `basic-auth-user` and `basic-auth-password`.
 
+**To create the secrets there are two options:**
+
+Option 1 (simpler, but doesn't works under Windows and docker complains about security)
+```bash
+echo myusername | docker secret create basic-auth-user -
+echo secr3t | docker secret create basic-auth-password -
+```
+
+Option 2 (safer, must work under any OS)
+```bash
+echo myusername > myusername.txt
```

Comment:
Prepend lines that show commands with `$ `
merge the two commands as we do elsewhere 
```
command1 && \
   command2
```

Reply:
Thanks for pointing these suggestions, this is my fist contibution. Will preppend the commands with ```$```, but prefer to leave each command separated beacuse ```&&`` works in Linux/Bash but not in Windows/Cmd.